src/test/java/org/magicdgs/readtools/tools/barcodes/AssignReadGroupByBarcodeIntegrationTest.java

```diff
@@ -180,7 +180,7 @@ public void testAddReadGroupByBarcodeOldToolConcordance(final String testName,
         final File expextedFilePrefix = getTestFile(testName);
 
         // add the outputs
-        final File actualOutputPrefix = new File(createTestTempDir(deprecatedTool), testName);
+        final File actualOutputPrefix = new File(createTempDir(deprecatedTool), testName);
```

Comment:
Is there a `BaseTest.createTempFile()` method to avoid the `new File` here (and below)?

Reply:
Cannot remove - here we create a prefix and not a file.